### ISSUE TITLE
Build for Apple Silicon and release via homebrew

### ISSUE THREAD

**Issue by hollesse** [remotemobprogramming/mob#352]
We should build an additional version for Apple Silicon and then release this via homebrew as well.

**Comment by ccoltx** [comment]
This would be great.

**Comment by hollesse** [comment]
I had a first look on this issue. I do know how to adjust the release process to get an arm executable. 

The point I was struggling with is homebrew as I never wrote a homebrew formulae before. I look for a way to get two executables into the formulae and even found one with two different urls and shas by using the on_intel and on_arm cases. But then I had a look at the PR bump feature. There I looked who to replace these two urls. I found an issue in the homebrew project which told that there shouldn't be multiple urls. So as I understood we are using homebrew the wrong way right now. The url should show to the source code which should be compiled and installed on the machine. Then we also would just have one url. And the our formulae would also work on Linux as you could also use homebrew on Linux.

Therefore the next steps are to add the arm build to the release pipeline such that people can download it directly if they want to. Afterwards we should rewrite the homebrew formulae to use the source code and compile it on the users machine. Therefore it would be good to find a formulae which is kind of similar to mob.

**Comment by hollesse** [comment]
Okay. I got a simple solution which i really like. I did not know, that there are universal apps for macOS which contain amd and arm version of the apps. I found out how to create such a universal app, which we can use in our build step. Then we do not need to adjust the homebrew formula. But I think in the end it makes also sense to do this, as linux user could then also use homebrew to install mob. But this is something for another issue.
